src/test/groovy/com/rabbitmq/http/client/ClientSpec.groovy

```diff
@@ -1156,6 +1156,33 @@ class ClientSpec extends Specification {
 
   }
 
+  def "PUT /api/operator-policies/{vhost}/{name}"() {
+    given: "vhost / and definition"
+    def v = "/"
+    def d = new HashMap<String, Object>()
+    d.put("ha-mode", "all")
```

Comment:
Please replace this with something like queue length limit. [Classic mirrored queues are going away completely in the next 3-9 months](https://rabbitmq.com/ha.html).